doc/release-notes.md

```diff
@@ -63,6 +63,14 @@ P2P and network changes
 
 Updated RPCs
 ------------
+- The top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and `descendantfees`
+  returned by RPCs `getmempoolentry`,`getrawmempool(verbose=true)`,
+  `getmempoolancestors(verbose=true)` and `getmempooldescendants(verbose=true)`
+  are now deprecated and will be removed in the next major version (use
+  `-deprecated=fees` if needed in this version). The same fee rates, all denominated
+  in BTC/kvB, can be accessed through the `fees` object in the result. Note: deprecated
+  fields `ancestorfees` and `descendantfees` are denominated in sat/vB, whereas all
+  fields in the `fees` object are denomainted in BTC/kvB (#22689)
```

Comment:
```suggestion
  fields in the `fees` object are denominated in BTC/kvB (#22689)
```

perhaps `s/Note:/Warning:`

Reply:
oof, good catch. i need to enable spell-check for vim :roll_eyes: